dockerfiles/remote-plugin-kubernetes-tooling-1.0.0/Dockerfile

```diff
@@ -31,4 +31,6 @@ COPY --from=endpoint /etc/passwd /etc/passwd
 COPY --from=endpoint /etc/group /etc/group
 COPY --from=endpoint /entrypoint.sh /entrypoint.sh
 
+RUN chmod g+w /home/theia
```

Comment:
it doesn't need to be `-R` ?

Reply:
no, buildah creates a new folder in user home dir